packages/plugin-network-breadcrumbs/network-breadcrumbs.js

```diff
@@ -120,11 +120,21 @@ const monkeyPatchFetch = () => {
 
   const oldFetch = win.fetch
   win.fetch = function fetch (...args) {
-    let [url, options] = args
+    const [urlOrRequest, options] = args
+
     let method = 'GET'
-    if (options && options.method) {
-      method = options.method
+    let url
+
+    if (typeof urlOrRequest === 'string') {
+      url = urlOrRequest
+      if (options && options.method) {
+        method = options.method
+      }
+    } else {
+      url = urlOrRequest.url
+      method = urlOrRequest.method
```

Comment:
According to [MDN's `fetch()` docs](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) the second parameter `options` can be an object containing options such as `method`.

I tried it in Firefox and it seems that `options.method` takes precedence over `urlOrRequest.method`, so that should be accounted for here.